src/edfi-performance-test/edfi_performance_test/__main__.py

```diff
@@ -48,21 +52,39 @@ def _configure_logging(configuration: MainArguments) -> None:
     _redefine_debug_as_verbose(configuration, "urllib3.connectionpool")
 
 
+def _url_reachable(url):
```

Comment:
@JBrenesSimpat could you move this to logic to a helper class and call the method from inside the 'run' method?